src/tests/mod.rs

```diff
@@ -19,7 +19,7 @@ mod main_tests {
     #[test]
     pub fn tests() {
         lindrustinit();
-        crate::lib_fs_utils::lind_deltree({CAGE_TABLE.read().unwrap().get(&1).unwrap()}, "/");
+        crate::lib_fs_utils::lind_deltree(CAGE_TABLE.read().unwrap().get(&1).unwrap(), "/");
         lindrustfinalize();
 
         println!("FS TESTS");
```

Comment:
Not necessary

Reply:
I thought this was for the fs tests so that we could count the files that we created. I removed the curly braces because they're unnecessary and add clutter, but I think this line is necessary.

Reply:
Does this delete the entire filesystem whenever tests are run?

Reply:
Yes — otherwise the persisted file system screws with the files and tests

Reply:
We discussed this over the summer. This won't clobber the lind file system, just the dummy one set up within the rust dirs for testing.